os/arch/arm/Kconfig

```diff
@@ -25,12 +25,26 @@ config ARCH_CHIP_S5J
 	select ARMV7R_MEMINIT
 	---help---
 		Samsung IoT SoC architectures (ARM Cortex R)
+
+config ARCH_CHIP_BCM4390X
+	bool "CY 4390X series"
+	select ARCH_CORTEXR4
+	select ARCH_HAVE_MPU
+	select ARMV7M_MPU
+	select ARMV7M_MPU_NREGIONS
+	select ARCH_HAVE_LOWVECTORS
+	select ARCH_LOWVECTORS
+	select ARM_UNWIND
```

Comment:
is it necessary to select this option ARM_UNWIND  by default ?

Reply:
Yes. correct. If not, Jtag reset command is not executed. 

BR, HJ Lee.